pkg/storage/etcd/v3_helper.go

```diff
@@ -34,6 +33,7 @@ import (
 
 var (
 	ErrNotFound = errors.New("etcd: key not found")
+	ErrTestFail = errors.New("etcd: test failed")
```

Comment:
Can you please change it to:
"etcd: compare failed"
?

"test failed" is suggesting that it is test-related, which it isn't